Menuetti/Controllers/RecipesController.cs

```diff
@@ -139,38 +131,16 @@ public IActionResult Create()
             }
             else
             {
-                List<Ingredientti> items = new List<Ingredientti>();
-
-                using (StreamReader r = new StreamReader("ingredients.json"))
-                {
-                    string ingredients = r.ReadToEnd(); // Read ingredients.json
-                    string ingredientsEdited = ingredients.Remove(ingredients.Length - 1).ToString(); // Remove the final "]" from the ingredients.json file to prepare the string for concatenation
-                    string json = string.Concat(ingredientsEdited, ",", AddedIngredients().Substring(1).ToString()); // Concatenate the above ingredients-string with a "," and an addedIngredients string with the initial "[" removed
-                    items = JsonConvert.DeserializeObject<List<Ingredientti>>(json).OrderBy(t => t.name.fi).ToList(); // Create a list of ingredients and added sorted alphabetically by the name
-                }
-
-                TempData["ingredients"] = items;
-
+                TempData["ingredients"] = GetIngredientList();
```

Comment:
I'd like to discuss the changes in order to get better understanding of the changes